common/ApplicationProcess.asn1

```diff
@@ -22,9 +22,10 @@
 --!
 
 ApplicationProcess DEFINITIONS AUTOMATIC TAGS ::= BEGIN
-
 EXPORTS ALL;
+IMPORTS
+    PUSC-UINT FROM BasicTypes;
 
-ApplicationProcess-ID ::= INTEGER((1)) -- TODO
+ApplicationProcess-ID ::= PUSC-UINT -- TODO
```

Comment:
do not use `BasicTypes` for now - we need to overhaul them :)

Reply:
The sooner the better :)